content/ch-quantum-information/density-matrix.ipynb

```diff
@@ -0,0 +1,1074 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "source": [
+    "# The Density Matrix & Mixed States"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Density matrix, or density operator, is an alternative way of expressing quantum states. Unlike the state vector notation introduced in the [Representing Qubit States](../ch-states/representing-qubit-states.ipynb) section, the density matrix provides a convinient and elegant way to introduce the notion of classical statistical distributions to quantum systems. For example, when working with real quantum computers, the state of a qubit is subject to the noise of its enviroment and to uncertainties generated by the qubit's control hardware. This state is therefore not only a function of its own quantum probability amplitudes, but it also depends on the probability distribution of the noise around it. We call these type of states \"mixed\" to distinguish them from the \"pure\" states that don't depend on these externalities. \n",
```

Comment:
> the probability distribution of the noise around it

I am not sure what you mean by this?

Reply:
OK. It seems the edits didn't help and this still isn't clear. I will rewrite it.